src/usd/sdf_usd_parser/sensor.cc

```diff
@@ -44,11 +44,14 @@ namespace usd
 
     // TODO(adlarkin) check units to make sure they match (no documented units for SDF)
```

Comment:
If the conversions below seem to be okay regarding units, can we remove this comment?